src/main/java/alfio/model/system/ConfigurationKeys.java

```diff
@@ -89,6 +89,7 @@
     OFFLINE_REMINDER_HOURS("How many hours before expiration should be sent a reminder e-mail for offline payments?", false, SettingCategory.PAYMENT_OFFLINE, ComponentType.TEXT, false, EnumSet.of(SYSTEM, ORGANIZATION, EVENT), true),
     BANK_ACCOUNT_NR("Bank Account number", false, SettingCategory.PAYMENT_OFFLINE, ComponentType.TEXT, false, EnumSet.of(SYSTEM, ORGANIZATION, EVENT), false),
     BANK_ACCOUNT_OWNER("Bank Account owner", false, SettingCategory.PAYMENT_OFFLINE, ComponentType.TEXTAREA, false, EnumSet.of(SYSTEM, ORGANIZATION, EVENT), true),
+    DISABLE_AUTOMATIC_REMOVAL_EXPIRED_OFFLINE_PAYMENT("Disable automatic removal of expired offline payments", false, SettingCategory.PAYMENT_OFFLINE, ComponentType.BOOLEAN, false, EnumSet.of(SYSTEM, ORGANIZATION, EVENT), true),
```

Comment:
what if we change this to: 
AUTOMATIC_REMOVAL_EXPIRED_OFFLINE_PAYMENT
description: Cancel Reservation automatically when payment is overdue
default: true
?